walletrpc/service.proto

```diff
@@ -101,6 +111,8 @@ service CompactTxStreamer {
 
     // Return the txids corresponding to the given t-address within the given block range
     rpc GetAddressTxids(TransparentAddressBlockFilter) returns (stream RawTransaction) {}
+    rpc GetAddressBalance(AddressList) returns (Balance) {}
```

Comment:
was this comment addressed? I think you made a good point here and it might be ideal to make it clear in code that this does not work for zaddrs. Perhaps we do the same as the `getAddresTxids` call and name the parameter in such a way as to make that distinction clear:
```
rpc GetAddressBalance(TransparentAddressList) returns (Balance) {}
rpc GetAddressBalanceStream(stream TransparentAddress) returns (Balance) {}
```
Personally, I think I prefer your suggestion of just calling these `GetTaddressBalance` and `GetTaddressBalanceStream`